src/Umbraco.Web.UI.Client/src/views/webhooks/webhooks.controller.js

```diff
@@ -68,8 +80,20 @@
     }
 
     function resolveTypeNames(webhook) {
-      const isContent = webhook.events[0].toLowerCase().includes("content");
-      const resource = isContent ? contentTypeResource : mediaTypeResource;
+      let resource;
+      switch (webhook.events[0].eventType.toLowerCase()) {
+        case "content":
+          resource = contentTypeResource;
+          break;
+        case "media":
+          resource = mediaTypeResource;
+          break;
+        case "member":
+          resource = memberTypeResource;
+          break;
+        default:
+          return;
+      }
```

Comment:
This code block duplicated within the same controller. Could do with a helper method instead.

Ideally we would create a resource that could be re-used in `webhooks/overlays/edit.controller.js` (where this code block also exists), but let's keep it simple for now 😄

Reply:
Done 🚀